packages/bpk-component-text/src/BpkText.scss

```diff
@@ -0,0 +1,27 @@
+@import '~bpk-mixins';
+
+.bpk-text {
```

Comment:
This reverted back to having `bpk-text` and the styles as modifiers because we need to remove margin for all text styles